src/shogun/statistical_testing/MMD.cpp

```diff
@@ -51,14 +51,17 @@ struct CMMD::Self
 		stype = DEFAULT_STYPE;
 		null_approximation_method = DEFAULT_NULL_APPROXIMATION_METHOD;
 		strategy=unique_ptr<CKernelSelectionStrategy>(new CKernelSelectionStrategy());
+		seed = DEFAULT_NUM_OF_SEED;
 	}
 
 	index_t num_null_samples;
 	EStatisticType stype;
 	ENullApproximationMethod null_approximation_method;
 	std::unique_ptr<CKernelSelectionStrategy> strategy;
+	int32_t seed;
 
 	static constexpr index_t DEFAULT_NUM_NULL_SAMPLES = 250;
+	static constexpr int32_t DEFAULT_NUM_OF_SEED = 12345;
```

Comment:
sure, so basically as I comment in https://github.com/shogun-toolbox/shogun/pull/3888#discussion_r125174002, it's about to set default seed of random as sg_rand->set_seed(12345). And it's going to set a fixed seed in https://github.com/shogun-toolbox/shogun/pull/3888/files#diff-e76beae1ecab5719393a08a37a5a5ef8R101 for CrossValidationMMD. But it's impossible to do that (give a fixed seed to CrossValidationMMD inside), so that's only way I can pass a fixed seed as parameter to it from outside.